generators/app/templates/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= _.kebabCase(appName) %></title>
```

Comment:
I think we don't need to worry about this, should be changed in any serious project anyway. I've let kebab-case because I delete the old index and didn't worry to see how it was.